frontend/packages/ux-editor/src/containers/DesignView/AddItem/AddItem.module.css

```
@@ -1,3 +1,15 @@
+.root {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  margin-left: 12px;
+  margin-right: 12px;
```

I believe you can use these tokens here.
```suggestion
  margin-left: var(--fds-spacing-3);
  margin-right: var(--fds-spacing-3);
```